lib/forcegraph.ts

```diff
@@ -42,64 +54,69 @@ export const ForceGraph = function (linkScale, sidebar) {
     draw.setMaxArea(canvas.width, canvas.height);
   }
 
-  function transformPosition(p) {
+  function transformPosition(p: { k: number; x: number; y: number }) {
+    // @ts-ignore
```

Comment:
To solve this ignore transform should be replaced with a new instance 

```
transform = new d3Zoom.ZoomTransform(p.k, p.x, p.y);
```

Reply:
I would love doing that but that breaks it even more, see the comments above.
Can you eventually show me exactly how you mean that at some point?